brambling/templates/brambling/forms/markdown/_js.html

```diff
@@ -0,0 +1,90 @@
+<script src="https://cdn.tinymce.com/4/tinymce.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/to-markdown/1.3.0/to-markdown.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/markdown-it/5.1.0/markdown-it.js"></script>
+<script>
+	$('[data-toggle="tab"]').click(function (e) {
+		e.preventDefault();
+		$(this).tab('show');
+	});
```

Comment:
I'm pretty sure, yeah. It was the old method of switching between WYSIWYG & markdown - but tinymce supersedes that.